src/patternfly/components/TextInputGroup/text-input-group.hbs

```diff
@@ -1,6 +1,16 @@
-<div class="pf-c-text-input-group{{#if text-input-group--modifier}} {{text-input-group--modifier}}{{/if}}"
+<div class="pf-c-text-input-group
+  {{#if text-input-group--modifier}}
+    {{text-input-group--modifier}}
+  {{/if}}
+  {{#if text-input-group--IsDisabled}}
+    pf-m-disabled
+  {{/if}}"
   {{#if text-input-group--attribute}}
     {{{text-input-group--attribute}}}
+  {{/if}}
+  {{#if text-input-group--IsDisabled}}
+    aria-label="Disabled text input group example"
```

Comment:
@jessiehuff can you confirm whether we need an `aria-label` on the outer container or just on the text input inside? I'm assuming the text input since this is designed to mimic the existing TextInput component.

Reply:
I'll go ahead and move this label to just the input itself.

Reply:
I agree, the `aria-label` makes the most sense on the text input like the TextInput component so that when the user gets to the input, it hears the label. That's where the screen reader would expect to interact with it. :)